thirdparty/README.md

```diff
@@ -556,8 +556,11 @@ File extracted from upstream release tarball:
 
 - All `.h` from `include/mbedtls/` to `thirdparty/mbedtls/include/mbedtls/`
   and all `.h` from `include/psa/` to `thirdparty/mbedtls/include/psa/`
-- All `.c` and `.h` from `library/` to `thirdparty/mbedtls/library/` except
-  for the `psa_*.c` source files
+- All `.c` and `.h` from `library/` to `thirdparty/mbedtls/library/`
+- From `library/` to `thirdparty/mbedtls/library/`:
+  - All `.c` and `.h` files
+  - Except `psa_*.c`, `bignum_mod.c`, `block_cipher.c`, `ecp_curves_new.c`,
```

Comment:
Actually we're now using the `psa_*.c` files after #96394, @Faless just omitted to update the instructions. Could you remove the exception for `psa_*.c` in this PR too?